os/net/lwip/configs/icmp6/Kconfig

```diff
@@ -3,7 +3,13 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
-if NET_IPv6_ICMP
+menu "ICMP6 Support"
+
+config NET_IPv6_ICMP
+	depends on NET_IPv6
+	default y
+	---help---
+		Build in support icmp for IPv6.
```

Comment:
Does this mean 
Buil**t**-in support **for** ICMP for IPv6 ?

Reply:
It's a mandatory option for IPv6 as described in this PR summary.